strongbox-parent/pom.xml

```diff
@@ -583,6 +583,39 @@
             </releases>
         </repository>
     </repositories>
+    <pluginRepositories>
```

Comment:
As discussed, this is already handled by the `settings.xml` file I showed you. You should be using that instead of defining it in the `pom.xml`-s. That's how we have it across the rest of the project.